packages/ts-example-selected-features/vite.config.js

```
@@ -14,39 +14,42 @@ See the License for the specific language governing permissions and
+
+import { failOnChunkSizeExceeded } from '../../scripts/vite/chunk-size-limit.mjs';
+import { maxGraphCodeSplitting } from '../../scripts/vite/maxgraph-chunk.mjs';
+
+// The maximum size of the maxgraph chunk, in kB. Declared once and passed to both the Vite warning and the blocking
+// check, so they cannot drift apart.
+const chunkSizeLimitInKB = 362;
 
 export default defineConfig(({ mode }) => {
+  // The analyzer slows the build down and starts a report server, so only analyze the production bundle when
+  // explicitly asked for it: see the 'build:analyze' script.
+  const isBundleAnalysisEnabled = mode === 'production' && process.env.ANALYZE === 'true';
```

ESLint

[error] 'process' is not defined.

(no-undef)